### ISSUE TITLE
Output of `kubectl describe pod` formats incorrectly

### ISSUE THREAD

**Issue by barrettj12** [kubernetes/kubernetes#112621]
**What happened**:
Output of `kubectl describe pod` formats incorrectly.
```console
$ microk8s kubectl describe pod -n controller-c2 controller-0
...
Events:
  Type     Reason            Age   From               Message
  ----     ------            ----  ----               -------
  Warning  FailedScheduling  79s   default-scheduler  running PreFilter plugin "VolumeBinding": %!!(MISSING)w(<nil>)
$
```
Probably a bug in the formatting code.

**What you expected to happen**:
Display the output more nicely, e.g.
```
running PreFilter plugin "VolumeBinding": nil
```

**How to reproduce it (as minimally and precisely as possible)**:

TBD

**Anything else we need to know?**:

**Environment**:
- Kubernetes client and server versions (use `kubectl version`):
  ```console
  $ microk8s kubectl version
  WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  
  Use --output=yaml|json to get the full version.
  Client Version: version.Info{Major:"1", Minor:"24+", GitVersion:"v1.24.4-2+2f38f78fa07274", 
  GitCommit:"2f38f78fa07274a290eb3b5802e82b1de6f70fa4", GitTreeState:"clean", BuildDate:"2022-08-22T10:27:57Z", GoVersion:"go1.18.5", Compiler:"gc", Platform:"linux/amd64"}
  Kustomize Version: v4.5.4
  Server Version: version.Info{Major:"1", Minor:"24+", GitVersion:"v1.24.4-2+2f38f78fa07274", 
  GitCommit:"2f38f78fa07274a290eb3b5802e82b1de6f70fa4", GitTreeState:"clean", BuildDate:"2022-08-22T10:25:23Z", GoVersion:"go1.18.5", Compiler:"gc", Platform:"linux/amd64"}
  ```
- Cloud provider or hardware configuration: *running microk8s on local machine*
- OS (e.g: `cat /etc/os-release`):
  ```console
  $ cat /etc/os-release
  NAME="Ubuntu"
  VERSION="20.04.4 LTS (Focal Fossa)"
  ID=ubuntu
  ID_LIKE=debian
  PRETTY_NAME="Ubuntu 20.04.4 LTS"
  VERSION_ID="20.04"
  HOME_URL="https://www.ubuntu.com/"
  SUPPORT_URL="https://help.ubuntu.com/"
  BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
  PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
  VERSION_CODENAME=focal
  UBUNTU_CODENAME=focal
  ```

**Comment by HirazawaUi** [comment]
I think I fixed the bug, waiting for the reviewer to review the PR, hope it helps you

**Comment by sanposhiho** [comment]
I believe this bug has been already fixed in https://github.com/kubernetes/kubernetes/pull/112029.

**Comment by HirazawaUi** [comment]
> I believe this bug has been already fixed in #112029.

Thanks for the reminder, I have closed my PR

**Comment by kerthcet** [comment]
Since it's not a critical bug just some outputs, no need to cherry pick, close then.
/close
